tests/rt_35d.conf

```diff
@@ -2,22 +2,22 @@
 # CPLD Benchmark 35d tests                                                                                                                                                        #
 ###############################################################################################################################################################################
 
-COMPILE | APP=S2SW SUITES=FV3_GFS_v16_coupled,FV3_GFS_v16_coupled_noahmp,FV3_GFS_v16_coupled_nsstNoahmp                       | + hera.intel orion.intel | fv3 |  |
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2012010100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2012040100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2012070100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2012100100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013010100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013040100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013070100
-#RUN     | cpld_bmarkfrac_wave_v16_35d                                                                                         |                          | fv3 |  | 2013100100
+#COMPILE | APP=S2SW SUITES=FV3_GFS_v16_coupled,FV3_GFS_v16_coupled_noahmp,FV3_GFS_v16_coupled_nsstNoahmp                   | + hera.intel orion.intel | fv3 |  |
```

Comment:
Maybe I missed this in the discussion today, why are these tests commented out and not removed?

Reply:
I retained them because we are still running the RT version of the benchmark v16 configuration (``cpld_bmark_wave_v16``).